From 984839295b4160a5859273399254225b9967c570 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sat, 18 Jun 2022 22:28:20 +0200
Subject: [PATCH] Resolve build failure on i686-linux

gcc had to say:
gzdoom-g4.8.0/src/common/engine/stats.h:83:13: error: 'CPU' was not declared in this scope
   83 |         if (CPU.bRDTSC)
---
 src/common/engine/stats.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/common/engine/stats.h b/src/common/engine/stats.h
index d6e976ea5..0794455d2 100644
--- a/src/common/engine/stats.h
+++ b/src/common/engine/stats.h
@@ -35,6 +35,9 @@
 #define __STATS_H__
 
 #include "zstring.h"
+#if defined __i386__
+#include "x86.h"
+#endif
 
 #if !defined _WIN32 && !defined __APPLE__
 
-- 
2.36.1

